emergency-services/dashboard/about.md

```diff
@@ -0,0 +1,5 @@
+{% raw %}
```

Comment:
Empty doc?

Reply:
Jacob and I discussed offline but commenting here for consistency. 

For the sidebar links to work, the gitbook plugin requires them to be linked to a live page. I wanted a Dashboard and Dash distinction in that sidebar to make it easy for customer to navigate the 2 sets of 911 api documentation, and to do that I needed a Dashboard and Dash landing page.

Awaiting a response from the 911 product teams for some guidance on what the content of these pages should be .